src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -20,10 +20,14 @@ export default class TranscriptArea {
   static #ENDPOINT_TRANSCRIPT = '/transcript';
   static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
   static #PARAM_ID = 'id';
+  static #TRANSCRIPT_ERROR_MESSAGE =
+      'Sorry, there is no transcript available for this lecture recording. :(';
 
+  static #hasTranscript = true;
```

Comment:
Is it possible for this to be set when you check if a transcript exists or not? If we start seeking before we know a transcript actually exists will there be an issue?

Reply:
I believe so. I'll update it to be set there when #286 is in master (PR that creates the scroll container). I added a TODO. There will be an error message displayed if it's seeked beforehand.

Reply:
Sounds good

Reply:
I ended up getting rid of the variable and checking if `currentTranscriptLine()` returns null instead because that should also handle the case where the transcript has not finished loading.